src/screens/CreatePost/index.js

```diff
@@ -713,9 +689,11 @@ const CreatePost = () => {
             setTypeUser={setTypeUser}
             isAnonymous={typeUser}
             anonUserInfo={anonUserInfo}
-            username={dataProfile.username ? dataProfile.username : 'Loading . . .'}
+            username={profile?.myProfile?.username}
             photo={
-              dataProfile.profile_pic_path ? {uri: dataProfile.profile_pic_path} : ProfileDefault
+              profile?.myProfile?.profile_pic_path
+                ? {uri: profile?.myProfile?.profile_pic_path}
+                : ProfileDefault
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be related to using the new `profile` context instead of the old `dataProfile` state. This could potentially improve performance by avoiding unnecessary re-renders, but it also introduces a tighter coupling between this component and the context provider. Make sure that the context provider is set up correctly and that it wraps this component in the component tree.
```diff
- username={dataProfile.username ? dataProfile.username : 'Loading . . .'}
- photo={
-  dataProfile.profile_pic_path ? {uri: dataProfile.profile_pic_path} : ProfileDefault
- }
+ username={profile?.myProfile?.username}
+ photo={
+   profile?.myProfile?.profile_pic_path
+     ? {uri: profile?.myProfile?.profile_pic_path}
+     : ProfileDefault
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->